tests/rptest/tests/cloud_topics/e2e_test.py

```diff
@@ -25,6 +25,12 @@
 from rptest.tests.end_to_end import EndToEndTest
 from rptest.util import Scale
 
+ALLOW_LIST = [
+    # TODO: remove once the problem is fixed
+    "Exception building and putting object",
+    "Error while sending request: Unfinished objects remain",
+]
+
 
 class EndToEndCloudTopicsBase(EndToEndTest):
```

Comment:
At this point do we even need to inherit from EndToEndTest? We are overriding the producer/consumer in all tests